preupg/xmlgen/xml_utils.py

```diff
@@ -209,16 +207,17 @@ def check_script_modification(self, key, k):
         self.update_values_list(self.rule, "{" + k + "}", key[k])
 
     def prepare_sections(self):
-        """The function prepares all tags needed for generation group.xml file."""
-        for main, self.keys in self.ini_files.items():
-            if main.endswith("group.ini"):
+        """The function prepares all tags needed for generation group.xml file.
+        """
+        for ini_filepath, ini_file_content in iter(self.ini_files.items()):
```

Comment:
Agree, I used http://legacy.python.org/dev/peps/pep-0469/ as a base for this change, but in case of _for_ cycle, the _iter_ is redundant. As @pirat89 mentioned, I used this style on many places I guess it would be easier to make a PR to deal with just usage of this redundant _iter_ :)